src/main/java/reposense/util/FileUtil.java

```diff
@@ -34,10 +37,52 @@
     // zip file which contains all the specified file types
     public static final String ZIP_FILE = "archive.zip";
 
+    public static final String SUMMARY_JSON_FILE = "summary.json";
+
     private static final Logger logger = LogsManager.getLogger(FileUtil.class);
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Zips only the relevant .JSON files
+     * @param configs Utilizes the relevant repo folders that are required to be zipped
+     */
+    public static void zipRelevantJsonFiles(List<RepoConfiguration> configs, Path sourceAndOutputPath) {
```

Comment:
The classes in util package including FileUtil are mean to be *tool* classes, so we do not put such specific methods in these classes. You should put methods that can be reused here. 

So, maybe you can have a method named `zipRelativeFiles` which takes in `a list of relative paths` and `sourceAndOutputPath` and zip the file. Then, you can generate the list of relative paths in another place and call this new method. The new method is more general and can be served as a `tool` method.

Above is just my suggestion, the key idea is not to make the method in `Utils` too specific.

Reply:
Alright noted. Have made changes to the code base and will re-request for review soon.